rules/TypeDeclaration/Rector/StmtsAwareInterface/DeclareStrictTypesRector.php

```diff
@@ -53,6 +53,11 @@ public function beforeTraverse(array $nodes): ?array
     {
         parent::beforeTraverse($nodes);
 
+        $filePath = $this->file->getFilePath();
+        if ($this->skipper->shouldSkipElementAndFilePath(self::class, $filePath)) {
+            return null;
+        }
```

Comment:
This should be moved to `AbstractRector::beforeTraverse()` or similar, as rector rule itself should not decide about the sip.

Reply:
It can't, as `AbstractRector::beforeTraverse()` on setup File object, and re-called early in current `parent::beforeTraverse()`, moving to AbstractRector will require `parent::beforeTraverse()` to not called early, but after.

Reply:
I see. Let's go for it now then.
Bbut in case more rules will need the skip fix, we'll have to move it to single place outside specific rules.